src/utils/types.ts

```diff
@@ -245,14 +252,19 @@ interface EvaluatorParameters {
 
 type ElementsById = { [templateElementId: string]: PageElement }
 
+interface ReviewInfo {
+  status: ReviewStatus
+}
```

Comment:
I don't think this would be required, on the new PR #411 the returned `reviewAssignment` already have a `review` field which is of existing type ReviewDetails and there is already the status plus the timeCreated, so that can be passed on to the next page. I think its better than having this here on the root of FullStructure

Reply:
sure we can get if from reviewAssignment, although I preferred it from FullStructure.

77d21e3a30980b13c803abcfff2d0d3cc99ec986

Reply:
actually it looks soo much better with AssignmentDetailsNEW, i'll change to that once it's merged